integration/cluster.go

```diff
@@ -830,3 +830,154 @@ type grpcAPI struct {
 	// Maintenance is the maintenance API for the client's connection.
 	Maintenance pb.MaintenanceClient
 }
+
+// WaitUntilLeaderChange waits until the member at given index gets leader change.
+func (c *ClusterV3) WaitUntilLeaderChange(t *testing.T, idx int) uint64 {
+	m := c.Members[idx]
+	oldLead := m.s.Lead()
+	timeout := m.s.Cfg.ReqTimeout()
+
+	donec, stopc := make(chan struct{}), make(chan struct{})
+	go func() {
+		defer close(donec)
+		for m.s.Lead() == oldLead {
+			select {
+			case <-stopc:
+				t.Fatalf("%s expected to leader change from %s (time-out %v)", m.s.ID(), types.ID(oldLead), timeout)
+			case <-time.After(time.Duration(m.s.Cfg.ElectionTicks) * time.Millisecond):
+			}
+		}
+	}()
+	select {
+	case <-donec:
+	case <-time.After(timeout):
+		close(stopc)
+	}
+
+	return m.s.Lead()
+}
+
+// RemoveMember removes and terminates the member with given id from cluster.
+func (c *ClusterV3) RemoveMember(t *testing.T, id uint64) {
```

Comment:
We can use current member add and remove method to cluster, but it will use v2 API.